File: Vowel and Consonant/Vowel_Consonant.c
#include<stdio.h>
int main(int argc, char** argv)
{
    char *ptr;
    int countVowel, countConsonant;
    ptr = argv[1];
    countVowel = countConsonant = 0;
    while(*ptr != '\0')
    {
        if(*ptr=='A' ||*ptr=='E' ||*ptr=='I' ||*ptr=='O' ||*ptr=='U' ||*ptr=='a' ||*ptr=='e' ||*ptr=='i' ||*ptr=='o' ||*ptr=='u')
            countVowel++;
        else
            countConsonant++;
        ptr++;
    }
    printf("Vowel = %d and Consonant = %d\n", countVowel, countConsonant);
    return 0;
}

File: Vowel and Consonant/Vowel_Consonant.java
class Vowel_Consonant {
    public static void main(String[] args) {
        String word = args[0];
        int countVowel, countConsonant;
        countVowel = countConsonant = 0;
        
        word = word.toLowerCase();
        
        for(int i=0; i<word.length(); i++){
            char ch = word.charAt(i);
            if(ch=='a' || ch=='e' || ch=='i' || ch=='o' || ch=='u')
                countVowel++;
            else
                countConsonant++;
        }

        System.out.println("Vowel = " + countVowel + " and Consonant = " + countConsonant);
    }
}
